src/btree/bt_sync_obsolete.c

```diff
@@ -786,11 +786,11 @@ __checkpoint_cleanup(void *arg)
 }
 
 /*
- * __wt_checkpoint_cleanup_create --
+ * __wt_obsolete_cleanup_create --
  *     Start the checkpoint cleanup thread.
```

Comment:
This comment needs updating/renaming too.

Reply:
Done.